api/src/test/java/org/openmrs/module/fhir2/api/translators/impl/ObservationTranslatorImplTest.java

```diff
@@ -199,21 +198,21 @@ public void toFhirResource_shouldConvertObsValueToValue() {
 	}
 	
 	@Test
-	public void toFhirResource_shouldConvertObsGroupToComponent() {
+	public void toFhirResource_shouldConvertObsGroupToHasMemberReferences() {
 		Obs observation = new Obs();
 		Obs childObs = new Obs();
 		childObs.setUuid(OBS_UUID);
 		observation.addGroupMember(childObs);
 		
-		Observation.ObservationComponentComponent component = new Observation.ObservationComponentComponent();
-		component.setId(OBS_UUID);
-		when(observationComponentTranslator.toFhirResource(childObs)).thenReturn(component);
+		Reference reference = new Reference();
+		reference.setId(OBS_UUID);
+		when(observationReferenceTranslator.toFhirResource(childObs)).thenReturn(reference);
 		
 		Observation result = observationTranslator.toFhirResource(observation);
 		
-		assertThat(result.getComponent(), notNullValue());
-		assertThat(result.getComponent().size(), equalTo(1));
-		assertThat(result.getComponent(), contains(hasProperty("id", equalTo(OBS_UUID))));
+		assertThat(result.getHasMember(), notNullValue());
+		assertThat(result.getHasMember().size(), equalTo(1));
+		assertThat(result.getHasMember(), contains(hasProperty("id", equalTo(OBS_UUID))));
```

Comment:
It's preferable, IMO to use `hasItem` instead of `contains`. The test here will also need to be updated to ensure that the `type` and `reference` values have been properly set.

Reply:
Here, wouldn't `hasProperty("id", ...)` have to be changed to something like `hasProperty("reference", "Observation/" + OBS_UUID)`, since `"id"` would correspond to the reference entity id and not the Obs id?

Reply:
Yes, I think that would do it